custom/ilsgateway/tanzania/reminders/__init__.py

```diff
@@ -171,6 +171,28 @@
 )
 
 
+EMG_ERROR = _(
+    "Your request was not formatted properly."
+    " Please try the request again, in the format: emg <product> <quantity> ..."
+)
+
+EMG_HELP = _(
```

Comment:
For all handlers in ILS when the user sends only keyword (in this case message will be "emg"), we send help message in response.